packages/utils-debugging-protocol-common/src/debugging-protocol-connector.ts

```diff
@@ -247,7 +246,7 @@ export class Connector implements IConnector {
 
         debug(`Error found loading ${resource}:\n%O`, params);
 
-        const element: AsyncHTMLElement = (await this.getElementFromRequest(params.requestId))!;
+        const element: AsyncHTMLElement = (await this.getElementFromRequest(params.requestId, this._dom))!;
```

Comment:
Ideally this should be `AsyncHTMLElement | null` (or simply omitted), but that requires updating the definition of `FetchError` to have `element: IAsyncHTMLElement | null`, which would be good to align it with the definition of `FetchEnd` that already allows `null`, but may be more than you want to do for this PR (as adding the `null` to `FetchError` could have ripple effects requiring changes in other packages).

Reply:
Yeah, I tried that before submitting the changes but it seemed to be more changes than needed. I'm working on fixing the perf budget tests but I can look into this later.

Reply:
I've fixed this.